Content.Server/Body/Components/RespiratorComponent.cs

```diff
@@ -29,6 +29,30 @@ public sealed partial class RespiratorComponent : Component
         [DataField]
         public float Saturation = 5.0f;
 
+        /// <summary>
+        /// <see cref="DamageSpecifier"/> of damage taken when breathing in gas at a high temperature.
+        /// </summary>
+        [DataField]
+        public DamageSpecifier HighTemperatureDamage = default!;
+
+        /// <summary>
+        /// <see cref="DamageSpecifier"/> of damage taken when breathing in gas at a low temperature.
+        /// </summary>
+        [DataField]
+        public DamageSpecifier LowTemperatureDamage = default!;
+
+        /// <summary>
+        ///     The maximum temperature of gas the entity can breathe before it starts taking damage.
+        /// </summary>
+        [DataField]
+        public float HighTemperatureDamageThreshold = 253.15f; // -20C
```

Comment:
This should the LowTemperatureDamageThreshold, right? It's lower than the other value.